### ISSUE TITLE
Test user authentication deployed on predev

### ISSUE THREAD

**Issue by tapaswenipathak** [OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io#174]
[Predev deployment](http://opensourcehelpcommunity-predev.herokuapp.com) now has user authentication enabled. We need to test it for different scenarios. While testing the errors and behavior is not consistent as what we can see when we tested internally.

As part of this issue, please go ahead and test predev deployment, specially user authentication and feel free to raise issues for bugs or enhancements.

**Comment by kurianbenoy** [comment]
I GOT an internal server error when signing up

**Comment by vaibhavsingh97** [comment]
@kurianbenoy Yes, It's because authentication try to send email but SMTP is not configured as email and password will be exposed to public. Check here:
https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/blob/4b8f7c3620497e1b15a56e4d7d988667b6a1f2ad/oshc/oshc/settings.py#L162-L163
So if you know a way that we can encrypt than please feel free to send PR.

**Comment by kurianbenoy** [comment]
@vaibhavsingh97  I have used a module named python-decouple to secure my SECRET KEY in django. I think this can be used in this case also

source : `https://simpleisbetterthancomplex.com/series/2017/10/16/a-complete-beginners-guide-to-django-part-7.html#project-settings`

**Comment by tapaswenipathak** [comment]
Please go ahead @kurianbenoy!

**Comment by vaibhavsingh97** [comment]
Awesome!! 😍😍

**Comment by kurianbenoy** [comment]
@tapasweni-pathak  and @vaibhavsingh97  I wont be able to work on the issue immediately. I will be available only from December 20(due to university examination)
